doc/configuration.md

```diff
@@ -18,6 +18,7 @@ The following options control the behavior of BuildCache:
 | `BUILDCACHE_DIR` | - | The cache root directory | `$HOME/.buildcache` |
 | `BUILDCACHE_DISABLE` | `disable` | Disable caching (bypass BuildCache) | false |
 | `BUILDCACHE_HARD_LINKS` | `hard_links` | Allow the use of hard links when caching | false |
+| `BUILDCACHE_HASH_EXTRA_FILES` | `hash_extra_files` | Extra file(s) whose content add to the hash | None |
```

Comment:
Grammar: "whose content **to** add to the hash".

I also think that more documentation is needed, e.g. when/how to use this option. An example would be good.

With #156 the plan is to add full documentation of each configuration parameter. I think we can start the work right away with this parameter and add something like this at the end of this document:

## BUILDCACHE_HASH_EXTRA_FILES

Bla-bla-bla...

Reply:
I added an explanation about two possible use cases.